bundles/org.openhab.binding.metofficedatahub/src/main/test/org/openhab/binding/metofficedatahub/internal/dto/responses/SiteApiFeatureCollectionTest.java

```diff
@@ -0,0 +1,2232 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.metofficedatahub.internal.dto.responses;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.junit.jupiter.api.Test;
+import org.openhab.binding.metofficedatahub.internal.MetOfficeDataHubBindingConstants;
+
+/**
+ * The {@link SiteApiFeatureCollectionTest} class implements unit test case for {@link SiteApiFeatureCollection}
+ *
+ * @author David Goodyear - Initial contribution
+ */
+@NonNullByDefault
+public class SiteApiFeatureCollectionTest {
+
+    public final static String siteApiDailyResponse = "{\n" +
```

Comment:
For readability and maintainability it is better to move the json to a seperate json file and read that file into a string on test run.

Reply:
Agreed, moved without using apache commons to reduce dependencies. If there's a simpler way please advise thank you.